docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### Duplicates Metrics
+
+This is the data file containing duplicates metrics for aligned CRAM files. Multiple reads that match at the same position in the genome are located and tagged as duplicate reads in the CRAM file, where duplicate reads are defined as originating from a single fragment of DNA.
+
+#### File Types
+
+| Filename Pattern          | Description                                | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------------- | ------------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
+
+#### File Types
+
+| Filename Pattern     | Description                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| -------------------- | ------------------------------------ | ------------- | ----------------------- | -------------------------- |
+| \*.ascat_metrics.tgz | Estimated by Sanger ASCAT CNV caller | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Cross Sample Contamination
+
+Data file containing cross sample contamination estimate.
+
+#### File Types
+
+| Filename Pattern             | Description                                                                                                                                     | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.contamination_metrics.tgz | Estimated by Sanger `verifyBamHomChk.pl` script, which provides information to determine whether the sample is possibly contaminated or swapped | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Genotyped Gender Metrics
```

Comment:
should this be `Genotyping Inferred Gender`

Reply:
Genotyped Gender Metrics is the datatype of the file, so I went with that for consistency.

Reply:
`Genotyping Inferred Gender` is the dataType, see here: https://song.rdpc.cancercollaboratory.org/studies/OCCAMS-GB/analysis

Reply:
oh i see - i was going off of this: https://wiki.oicr.on.ca/display/icgcargotech/File+dataType+specification

Reply:
changed.